frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -149,6 +149,8 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
               onEnter={() => {
                 setFindMoreOpen(true);
                 setFieldValue('surplusFilter', true);
+                setFieldValue('includeAllProperties', true);
```

Comment:
Will have to adjust this if we want them to be able to have this as false

Reply:
After closing the SPP form, can they then alter the agency and research?